test/sharness/Rules.mk

```diff
@@ -49,13 +49,16 @@ $(SHARNESS_$(d)): $(d) ALWAYS
 $(d)/deps: $(SHARNESS_$(d)) $$(DEPS_$(d)) # use second expansion so coverage can inject dependency
 .PHONY: $(d)/deps
 
-test_sharness_deps: $(d)/deps
+$(d)/sharness-ipfs:
+	$(MAKE) cmd/ipfs/ipfs PLUGIN_PATH=./system_plugins
```

Comment:
This is invoking makefile recursively, don't do that. I will take a look how to make it not recursive.